migrations/v2.js

```diff
@@ -0,0 +1,205 @@
+      )
+    );
+    if (!isValid) throw new Error('adapt-contrib-accordion - _graphic.src not added to every instance of accordion._items');
+    return true;
+  });
+
+  /**
+    * * Add a new field to course globals.
+    */
+  mutateContent('adapt-contrib-accordion - add globals ariaRegion attribute', async (content) => {
+    course = content.find(({ _type }) => _type === 'course');
+    courseAccordionGlobals = course._globals._components._accordion ?? {};
+    courseAccordionGlobals.ariaRegion = 'This component is an accordion comprised of collapsible content panels containing display text. Select the item titles to toggle the visibility of these content panels.';
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check globals ariaRegion attribute', async () => {
+    const isValid = courseAccordionGlobals.ariaRegion === 'This component is an accordion comprised of collapsible content panels containing display text. Select the item titles to toggle the visibility of these content panels.';
+    if (!isValid) throw new Error('Accordion globals ariaRegion attribute not added.');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-accordion - update to v2.0.0', { name: 'adapt-contrib-accordion', version: '2.0.0', framework: '>=2' });
+});
+
+describe('adapt-contrib-accordion - v2.0.0 > v2.0.4', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v2.0.0', { name: 'adapt-contrib-accordion', version: '<=2.0.4' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+   * * Add field to each item in a JSON array and set blank.
+   */
+  mutateContent('adapt-contrib-accordion - add accordion._items._classes attribute', async () => {
+    accordions.forEach(accordion => {
+      accordion._items.forEach(item => {
+        item._classes = '';
+      });
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion._items._classes attribute', async () => {
+    const isValid = accordions.every(accordion =>
+      accordion._items.every(item =>
+        item && item._classes !== undefined
+      )
+    );
+    if (!isValid) throw new Error('adapt-contrib-accordion - _classes not added to every instance of accordion._items');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-accordion - update to v2.0.4', { name: 'adapt-contrib-accordion', version: '2.0.4', framework: '>=2' });
+});
+
+describe('adapt-contrib-accordion - v2.0.4 > v2.0.5', async () => {
+  let accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v2.0.4', { name: 'adapt-contrib-accordion', version: '<=2.0.5' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
```

Comment:
```suggestion
    if (accordions.length > 0) return true;
```

or 
```suggestion
    if (accordions.length) return true;
```

Reply:
It would also work to just
```js
return accordions.length
```

Just to explain why you can't use `accordions` as a test, is because filter always returns an array, that array may be empty if it finds nothing, an array is always truthy, empty or not. You want to know if the array has items, which is `array.length` if 0 is returned that is a falsey, if >= that is a truthy.

Reply:
Looks a bit nicer doesn't it, will amend my suggestions/migration scripts to 
`return content.some(({ _component }) => _component === 'accordion');`

Reply:
@oliverfoster Just reviewing/testing that comment, while the suggestion is a bit nicer to look at, we need to set the accordion array to access further down to mutate and check, if we use the below we'll then need to additionally filter content. Might be a bit nicer finding them all and setting the array during the whereContent. Thoughts?

Reply:
In testing, it's generally really bad practice to carry data around outside of the tests. Each test should be able to run independently without impacting another test. 

In this case it would be fine, but it means that a person has to read the entire describe block to understand the mutations.

Note that the where bodies will run multiple times in order to ascertain if and when each describe should be included in the current applicable batch of scripts when running.